### ISSUE TITLE
Missing Spanish imperative affirmatives

### ISSUE THREAD

**Issue by Perspx** [SekouDiaoNlp/mlconjug3#48]
I discovered some missing Spanish conjugation results for the second-person singular imperative form of some  -ir and -er verbs; in my testing for ~3250 verbs I found missing results for:
- Regular verbs `equivaler`, `prevaler` and `valer`: should be `tu equivale` and `tu prevale`
- Irregular verbs `salir` and `sobresalir`: should be `tu sal` and `tu sobresal`

To test, run:

    import mlconjug3

    default_conjugator = mlconjug3.Conjugator("es")
    print(default_conjugator.conjugate("salir").conjug_info['Imperativo']['Imperativo Afirmativo'])

Note that for each verb the dictionary contains `('2s', None)`

**Comment by SekouDiaoNlp** [comment]
Hi @Perspx .

Thanks for submitting the bug request, I will investigate the issue and release a new version with the fix as soon as possible.

**Comment by SekouDiaoNlp** [comment]
Hi @Perspx I just pushed the fix for this issue and issue #47 on the master branch here on GitHub.

While I was investigating this issue, I discovered a new [issue](https://github.com/SekouDiaoNlp/mlconjug3/issues/50) where some Spanish verbs were missing some conjugated forms.

I will publish a fix for this new issue today or in the next couple of days and then I will release a new version of mlconjug3 on PyPi.

Thanks again for using mlconjug3 and for submitting bug reports.

Cheers,

SekouDiaoNlp.
